src/prefect/cli/run.py

```diff
@@ -1,42 +1,688 @@
+            "Received no options to look up the flow." + FLOW_LOOKUP_MSG
+        )
+    if "--id" in given_lookup_options and name:
+        raise ClickException(
+            "Received too many options to look up the flow; "
+            "cannot specifiy both `--name` and `--id`" + FLOW_LOOKUP_MSG
+        )
+    if len(given_lookup_options) > 1:
+        raise ClickException(
+            "Received too many options to look up the flow: "
+            f"{', '.join(given_lookup_options)}" + FLOW_LOOKUP_MSG
+        )
+
+    # Load parameters and context ------------------------------------------------------
+    context_dict = load_json_key_values(context_vars, "context")
+
+    file_params = {}
+    if param_file:
+
+        try:
+            with open(param_file) as fp:
+                file_params = json.load(fp)
+        except FileNotFoundError:
+            raise TerminalError(
+                f"Parameter file does not exist: {os.path.abspath(param_file)!r}"
+            )
+        except ValueError as exc:
+            raise TerminalError(
+                f"Failed to parse JSON at {os.path.abspath(param_file)!r}: {exc}"
+            )
+
+    cli_params = load_json_key_values(params, "parameter")
+    conflicting_keys = set(cli_params.keys()).intersection(file_params.keys())
+    if conflicting_keys:
+        quiet_echo(
+            "The following parameters were specified by file and CLI, the CLI value "
+            f"will be used: {conflicting_keys}"
+        )
+    params_dict = {**file_params, **cli_params}
+
+    # Local flow run -------------------------------------------------------------------
+
+    if path or module:
+        # We can load a flow for local execution immediately if given a path or module,
+        # otherwise, we'll lookup the flow then pull from storage for a local run
+        with try_error_done("Retrieving local flow...", quiet_echo, traceback=True):
+            flow = get_flow_from_path_or_module(path=path, module=module, name=name)
+
+        # Set the desired log level
+        if no_logs:
+            log_level = 100  # CRITICAL is 50 so this should do it
+
+        run_info = ""
+        if params_dict:
+            run_info += f"└── Parameters: {params_dict}\n"
+        if context_dict:
+            run_info += f"└── Context: {context_dict}\n"
+
+        if run_info:
+            quiet_echo("Configured local flow run")
+            quiet_echo(run_info, nl=False)
+
+        quiet_echo("Running flow locally...")
+        from prefect.utilities.logging import temporary_logger_config
+
+        with temporary_logger_config(
+            level=log_level,
+            stream_fmt="└── %(asctime)s | %(levelname)-7s | %(message)s",
+            stream_datefmt="%H:%M:%S",
+        ):
+            with prefect.context(**context_dict):
+                try:
+                    result_state = flow.run(parameters=params_dict)
+                except Exception as exc:
+                    quiet_echo()
```

Comment:
remove?

Reply:
This was to print a newline which may or may not be nice

Reply:
I'll add a statement